pkg/fanal/analyzer/language/golang/mod/mod.go

```diff
@@ -100,7 +100,7 @@ func (a *gomodAnalyzer) PostAnalyze(_ context.Context, input analyzer.PostAnalys
 	}
 
 	if err = a.fillAdditionalData(apps); err != nil {
-		return nil, xerrors.Errorf("unable to collect additional info: %w", err)
+		log.Logger.Warnf("unable to collect additional info: %s", err)
```

Comment:
The warning message should be capitalized, unlike error messages.
```suggestion
		log.Logger.Warnf("Unable to collect additional info: %s", err)
```